config.go

```diff
@@ -573,6 +632,11 @@ func loadConfig() (*config, []string, error) {
 		}
 	}
 
+	// Always prompt passphrase if stake mining is enabled
```

Comment:
Instead of modifying the config, it just leave the config as is and when cfg.EnableStakeMining is true it can just run the password prompt at startup anyways.